src/Sylius/Component/Core/OrderProcessing/OrderExchangeRateAndCurrencyUpdater.php

```diff
@@ -51,5 +61,7 @@ public function update(OrderInterface $order)
 
         $order->setCurrencyCode($currency->getCode());
         $order->setExchangeRate($currency->getExchangeRate());
+
+        $this->cartRepository->add($order);
```

Comment:
It's `orderRepository`, but I think we should use `entityManager->flush` here, since we don't add the order but only update it.